src/SFA.DAS.EAS.Support.Web.Tests/Extensions/WhenTestingUrlHelperExtensions.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Web;
+using System.Web.Mvc;
+using System.Web.Routing;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EAS.Support.ApplicationServices.Models;
+using SFA.DAS.EAS.Support.ApplicationServices.Services;
+using SFA.DAS.EAS.Support.Infrastructure.Settings;
+using SFA.DAS.EAS.Support.Web.Configuration;
+using SFA.DAS.EAS.Support.Web.Controllers;
+using SFA.DAS.EAS.Support.Web.Extensions;
+using SFA.DAS.EAS.Support.Web.Services;
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.Support.Shared.SiteConnection;
+
+namespace SFA.DAS.EAS.Support.Web.Tests.Extensions
+{
+    [TestFixture]
+    public class WhenTestingUrlHelperExtensions
+    {
+        private WebConfiguration _configuration;
+        private Mock<IDependencyResolver> _dependancyResolver;
+        protected AccountController Controller;
+        private Mock<IAccountHandler> _accountHandler;
+        private Mock<IPayeLevySubmissionsHandler> _payeLevySubmissionsHandler;
+        private Mock<ILog> _log;
+        private Mock<IPayeLevyMapper> _payeLevyMapper;
+        private Mock<HttpContextBase> _httpContextMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            _accountHandler=new Mock<IAccountHandler>();
```

Comment:
We don't need these mocks (e.g. IAccountHandler) for the unit test or even a Controller at all.  Just create a new instance of a UrlHelper passing in the mocked RequestContext.

Reply:
cleaned up